commons/mysql/files/execute.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Please double-check all the lines referencing mysql if it works after you change the name of db to mariadb.